control-base/src/main/java/fi/nls/oskari/control/layer/GetWFSLayerFieldsHandler.java

```diff
@@ -50,6 +52,7 @@ private JSONObject getLayerFields(OskariLayer layer) throws ActionException {
             JSONObject fields = WFSGetLayerFields.getLayerFields(layer);
             JSONObject locale = getFieldsLocale(layer);
             fields.putOpt("locale", locale);
+            fields.putOpt("filter", getFieldsFilter(layer));
```

Comment:
Should we rename this to something like `visibleAttributes` now that we also have filter in another meaning?

Reply:
Renamed `filter` to `selection`. Don't know if it's better but I think that it's more like featuredata bundle to decide that properties which aren't selected is hidden for default. User then may click all visible and array isn't updated. Also it's used for ordering. Also renamed `attributes` to `types`. Maybe action route could be renamed as `GetWFSPropertiesInfo`, `GetWFSMetadata`, `GetWFSLayerInfo` etc.